go/common/gethapi/transaction_args.go

```diff
@@ -0,0 +1,299 @@
+package gethapi
```

Comment:
there should be a comment on this file saying that it's duplicating file x from geth

Reply:
Good point - Added 👍